strictdoc/export/html/_static/controllers/autocompletable_field_controller.js

```diff
@@ -46,6 +46,19 @@
         this.close()
       });
 
+      autocompletable.addEventListener("click", (event) => {      
+        /* toggle show / hide results */
```

Comment:
Let's capitalize to Toggle and end the comment with a dot. See my other comment about the traceable dev guide.